youtube_dl/extractor/neteasemusic.py

```diff
@@ -84,7 +84,8 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'id': '32102397',
             'ext': 'mp3',
             'title': 'Bad Blood (feat. Kendrick Lamar)',
-            'creator': 'Taylor Swift / Kendrick Lamar',
```

Comment:
So, are you suggesting that,

for tracks with single artist, the information should be:

```
'creator': 'A',
'artist': 'A',
```

and for tracks with multiple artists, the information should be

```
'creator': 'A',
'artist': 'A / B'
```

?

That's seems redundant.

---

Technically speaking, as `creator` is `The main artist who created the video` and `artist` is `Artist(s) of the track`, the `creator` field should be extracted from `artist` field automatically by youtube-dl core.